web/core/store/user/index.ts

```diff
@@ -229,6 +231,26 @@ export class UserStore implements IUserStore {
     this.store.resetOnSignOut();
   };
 
+  /**
+   * @description returns projects where user has permissions
+   * @returns {{[projectId: string]: number} || null}
+   */
+  get getProjectsWithPermissions() {
```

Comment:
get implies a function call. Can you update the name.
Also, all projects will have some permission. Here we are getting where the user is atleast a member.

Can you ensure the name reflects that.